public/achievementList.php

```diff
@@ -109,6 +112,7 @@
         $sort6 = ($sortBy == 6) ? 16 : 6;
         $sort7 = ($sortBy == 17) ? 7 : 17;
         $sort8 = ($sortBy == 18) ? 8 : 18;
+        $sort9 = ($sortBy == 19) ? 20 : 19;
```

Comment:
This should be 9 and 19 or the modulus 10 logic doesn't work, which causes the `*` to not appear when sorting by awarded ascending.

Reply:
Thanks, I didn't recognize this pattern but that totally makes sense. Should now be addressed.